app/controllers/api/v1/metrics_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
+    class MetricsController < ApiController
+      def create
+        # TODO: Find out how to get the branches without an user
+        BranchMetricUpdater.perform_async(User.first.id, project.id, metrics_params)
```

Comment:
I would add basic param validation before the async worker is called. In case params are not valid and the client never finds out